Shared/Samples/Display map from portal item/DisplayMapFromPortalItemView.swift

```diff
@@ -0,0 +1,103 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct DisplayMapFromPortalItemView: View {
+    /// The current portal item map.
+    @State private var currentMap = mapOptions.first!
+    
+    /// A map to display on the screen.
+    @State private var map = Map(item: mapOptions.first!.portalItem)
+    
+    /// The different map options the user can choose from.
+    private static let mapOptions = [
+        PortalItemMap(title: "Terrestrial Ecosystems of the World",
+                      id: .terrestrialEcosystems),
+        PortalItemMap(title: "Recent Hurricanes, Cyclones and Typhoons",
+                      id: .hurricanesCyclonesTyphoons),
+        PortalItemMap(title: "Geology of United States",
+                      id: .usGeology)
+    ]
+    
+    var body: some View {
+        MapView(map: map)
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Picker("", selection: $currentMap) {
```

Comment:
I'm OK with what we have now. If you want to display a consistent button label, you can wrap the picker in a menu like this

![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/81c9b6a2-bc7e-4798-b41d-8551a5289dda)


```swift
Menu("Choose Map") {
    Picker("", selection: $currentMap) {
        ForEach(DisplayMapFromPortalItemView.mapOptions) { mapOption in
            Text(mapOption.title).tag(mapOption)
        }
    }
    .onChange(of: currentMap) { _ in
        map = Map(item: currentMap.portalItem)
    }
}
```